agent/containers/images/Makefile

```diff
@@ -1,16 +1,46 @@
 # Base Makefile for building all images and tagging them
 
+# List of external Make targets:
+#
+#   FIXME:  Are there any missing from this list?  Are there any here which can/should be removed (e.g., the last two)??
```

Comment:
Are you referring to the "all-tags" and "all-dockerfiles"? Yeah, I don't really know why we'd need these; perhaps we could generate them for validation, but is that really worthwhile?

Reply:
I guess the idea is to be able to constuct pieces that one could use in a run "by hand", e.g. for debugging purposes. I vote we keep them.